rest-server/admin/src/main/scala/com/actionml/admin/MongoAdministrator.scala

```diff
@@ -144,10 +144,20 @@ class MongoAdministrator extends Administrator with JsonSupport {
 
   override def systemInfo(): Validated[ValidateError, Response] = {
     logger.trace("Getting Harness system info")
-    case class sysInfo(version: String = "0.5.0-SNAPSHOT")
-    val info = com.actionml.admin.BuildInfo
-    val ver = info.version
-    Valid(Comment(s"Version: ${com.actionml.admin.BuildInfo.version}"))
+    // todo: do we want to check connectons to services here?
+    Valid(SystemInfo(
+      buildVersion = com.actionml.admin.BuildInfo.version,
+      gitBranch = Properties.envOrElse("BRANCH", "No branch detected in env." ),
+      gitHash = Properties.envOrElse("GIT_HASH", "No short commit number detected in env." ),
+      harnessURI = Properties.envOrElse("HARNESS_URI", "No URI set, using host and port" ),
+      harnessHost = Properties.envOrElse("HARNESS_HOST", "No URI set, using host and port" ),
+      harnessPort = Properties.envOrElse("HARNESS_PORT", "No URI set, using host and port" ),
+      mongoURI = Properties.envOrElse("MONGO_URI", "No URI set" ),
```

Comment:
maybe it's time to put all those constants like "MONGO_URI" somewhere...

Reply:
@andreyshikov yeah, I'm removing the akka conf for `rest-server` and deriving everything from one HARNESS_URI that can be set in the env. See changed to

Reply:
If this looks ok I'm going to move all addresses to URIs and remove some akka conf